assets/package-lock.json

```diff
@@ -2967,6 +2988,12 @@
         "is-symbol": "^1.0.2"
       }
     },
+    "escalade": {
```

Comment:
https://www.npmjs.com/package/escalade is replacing `find-up`.
I don't see a need for either of them in our project ...
but that's what you get with Node.js dependencies ...